Network Activity/DNS Activity/Zeek/dns_log/README.md

```diff
@@ -0,0 +1,57 @@
+# Event Dossier: Zeek dns.log
+### NOERROR Query Results
+- **Description**: Translates a Zeek query log to OCSF. 
+- **Event References**:
+  - https://docs.zeek.org/en/master/logs/dns.html
+  - https://schema.ocsf.io/1.0.0-rc.3/objects/dns_query
+  
+ ### OCSF Version: 1.0.0-rc.2
+ - `category_uid`: `4`
+ - `category_name`: `Network Activity`
+ - `class_uid`: `4003`
+ - `class_name`: `DNS Activity`
+ - `metadata.profiles`: `[security_control]`
+ - `metadata.product.name`: `Zeek`
+ - `metadata.product.feature.name`: `dns.log`
+ - `metadata.product.vendor_name`: `Zeek`
+ - `severity`: `Informational`
+ - `severity_id`: `1`
+
+ ### Mapping:
+ - This does not reflect any transformations or evaluations of the data. Some data evaluation and transformation will be necessary for a correct representation in OCSF that matches all requirements.
+
+ - Any field not present in an explicit mapping will be moved to the unmapped object.
+
+| OCSF                          | Raw             |
+| ----------------------------- | --------------- |
+|'time'                         |'ts'             |
+|'uuid'                         |'uid'            |
```

Comment:
`uuid` is not directly available in the class. `metadata.uid` would be an appropriate destination here